amqp.reflect/setup.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
 set -ex
 
+# Verify zilla:develop-SNAPSHOT image already available locally
+docker image inspect ghcr.io/aklivity/zilla:develop-SNAPSHOT --format 'Image Found {{.RepoTags}}'
+
 # Install Zilla to the Kubernetes cluster with helm and wait for the pod to start up
 ZILLA_CHART=oci://ghcr.io/aklivity/charts/zilla
-helm install zilla-amqp-reflect $ZILLA_CHART --namespace zilla-amqp-reflect --create-namespace --wait \
+NAMESPACE=zilla-amqp-reflect
+helm upgrade --install zilla $ZILLA_CHART --namespace $NAMESPACE --create-namespace --wait \
     --values values.yaml \
     --set-file zilla\\.yaml=zilla.yaml \
     --set-file secrets.tls.data.localhost\\.p12=tls/localhost.p12
 
 # Start port forwarding
-kubectl port-forward --namespace zilla-amqp-reflect service/zilla-amqp-reflect 5671 5672 > /tmp/kubectl-zilla.log 2>&1 &
-until nc -z localhost 5671; do sleep 1; done
+kubectl port-forward --namespace $NAMESPACE service/zilla 7171 7172 > /tmp/kubectl-zilla.log 2>&1 &
+until nc -z localhost 7171; do sleep 1; done
```

Comment:
5671 or 7171? is it a typo or did we change to ports here? 🤔

Reply:
These are the updated ports to use `71` as a prefix given `7114` is the default "ZILA" port.